examples/testSketch/testSketch.ino

```diff
@@ -0,0 +1,18 @@
+
+double number;
```

Comment:
Later we will directly use tests and library/examples.
The built-in examples are not available in the arduino-cli :(